model/models.js

```diff
@@ -50,4 +50,52 @@ const fetchReviewById = (reviewId) =>
         return Promise.reject({status: 400, message: 'Invalid input!'});
     
 }
-module.exports = { fetchAllCategories, fetchAllReviews, fetchReviewById };
\ No newline at end of file
+
+const addComment = (reviewId, body) =>
```

Comment:
We have nested .then blocks here. One of main benefits of promises is that it allows to avoid nesting asynchronous operations. Please amend this before merging.

Reply:
I've removed the nested .then blocks and used promise.all. thanks!